{
    "metadata": {},
    "outputs": [
        "\u521b\u5efa\u6d4b\u8bd5\u76ee\u5f55 plugin_test",
        "\u9879\u76ee nonebot-plugin_orangedice \u521b\u5efa\u6210\u529f\u3002",
        "    \u001b[39;1mVirtualenv\u001b[39;22m",
        "    \u001b[34mPython\u001b[39m:         \u001b[32m3.11.10\u001b[39m",
        "    \u001b[34mImplementation\u001b[39m: \u001b[32mCPython\u001b[39m",
        "    \u001b[34mPath\u001b[39m:           \u001b[32mNA\u001b[39m",
        "    \u001b[34mExecutable\u001b[39m:     \u001b[32mNA\u001b[39m",
        "    ",
        "    \u001b[39;1mBase\u001b[39;22m",
        "    \u001b[34mPlatform\u001b[39m:   \u001b[32mlinux\u001b[39m",
        "    \u001b[34mOS\u001b[39m:         \u001b[32mposix\u001b[39m",
        "    \u001b[34mPython\u001b[39m:     \u001b[32m3.11.10\u001b[39m",
        "    \u001b[34mPath\u001b[39m:       \u001b[32m/usr/local\u001b[39m",
        "    \u001b[34mExecutable\u001b[39m: \u001b[32m/usr/local/bin/python3.11\u001b[39m",
        "    Using version ^0.3.9 for nonebot-plugin-orangedice",
        "    ",
        "    Updating dependencies",
        "    Resolving dependencies...",
        "    ",
        "    Package operations: 18 installs, 0 updates, 0 removals",
        "    ",
        "      - Installing typing-extensions (4.12.2)",
        "      - Installing annotated-types (0.7.0)",
        "      - Installing idna (3.10)",
        "      - Installing multidict (6.1.0)",
        "      - Installing propcache (0.2.0)",
        "      - Installing pydantic-core (2.23.4)",
        "      - Installing greenlet (3.1.1)",
        "      - Installing loguru (0.7.2)",
        "      - Installing pydantic (2.9.2)",
        "      - Installing pygtrie (2.5.0)",
        "      - Installing python-dotenv (1.0.1)",
        "      - Installing yarl (1.14.0)",
        "      - Installing msgpack (1.1.0)",
        "      - Installing nonebot2 (2.3.3)",
        "      - Installing sqlalchemy (2.0.35)",
        "      - Installing nonebot-adapter-onebot (2.4.5)",
        "      - Installing sqlmodel (0.0.22)",
        "      - Installing nonebot-plugin-orangedice (0.3.9)",
        "    ",
        "    Writing lock file",
        "\u63d2\u4ef6 nonebot-plugin_orangedice \u4f9d\u8d56\u7684\u63d2\u4ef6\u5982\u4e0b\uff1a",
        "    nonebot_plugin_orangedice",
        "\u63d2\u4ef6 nonebot-plugin_orangedice \u7684\u4fe1\u606f\u5982\u4e0b\uff1a",
        "    name         : nonebot-plugin-orangedice      ",
        "     version      : 0.3.9                          ",
        "     description  : A COC dice plugin for nonebot2 ",
        "    ",
        "    dependencies",
        "     - nonebot-adapter-onebot >=2.2.3",
        "     - nonebot2 >=2.0.1",
        "     - pydantic >=1.9.2",
        "     - sqlmodel >=0.0.8",
        "\u63d2\u4ef6 nonebot_plugin_orangedice \u52a0\u8f7d\u51fa\u9519\uff1a",
        "    \u001b[32m10-10 14:22:11\u001b[0m [\u001b[32m\u001b[1mSUCCESS\u001b[0m] \u001b[36m\u001b[4mnonebot\u001b[0m\u001b[36m\u001b[0m | NoneBot is initializing...",
        "    \u001b[32m10-10 14:22:11\u001b[0m [\u001b[1mINFO\u001b[0m] \u001b[36m\u001b[4mnonebot\u001b[0m\u001b[36m\u001b[0m | Current \u001b[33m\u001b[1mEnv: prod\u001b[0m\u001b[33m\u001b[0m",
        "    \u001b[32m10-10 14:22:12\u001b[0m [\u001b[31m\u001b[1mERROR\u001b[0m] \u001b[36m\u001b[4mnonebot\u001b[0m\u001b[36m\u001b[0m | \u001b[31m\u001b[48;2;248;187;208mFailed to import \"nonebot_plugin_orangedice\"\u001b[0m\u001b[31m\u001b[0m",
        "    \u001b[33m\u001b[1mTraceback (most recent call last):\u001b[0m",
        "      File \"\u001b[32m/tmp/plugin_test/nonebot-plugin_orangedice-nonebot_plugin_orangedice/\u001b[0m\u001b[32m\u001b[1mrunner.py\u001b[0m\", line \u001b[33m16\u001b[0m, in \u001b[35m<module>\u001b[0m",
        "        \u001b[1mplugin\u001b[0m \u001b[35m\u001b[1m=\u001b[0m \u001b[1mload_plugin\u001b[0m\u001b[1m(\u001b[0m\u001b[36m\"nonebot_plugin_orangedice\"\u001b[0m\u001b[1m)\u001b[0m",
        "      File \"/tmp/plugin_test/nonebot-plugin_orangedice-nonebot_plugin_orangedice/.venv/lib/python3.11/site-packages/nonebot/plugin/load.py\", line 40, in load_plugin",
        "        return manager.load_plugin(module_path)",
        "    > File \"/tmp/plugin_test/nonebot-plugin_orangedice-nonebot_plugin_orangedice/.venv/lib/python3.11/site-packages/nonebot/plugin/manager.py\", line 167, in load_plugin",
        "        module = importlib.import_module(self._third_party_plugin_ids[name])",
        "      File \"/usr/local/lib/python3.11/importlib/__init__.py\", line 126, in import_module",
        "        return _bootstrap._gcd_import(name[level:], package, level)",
        "      File \"<frozen importlib._bootstrap>\", line 1204, in _gcd_import",
        "      File \"<frozen importlib._bootstrap>\", line 1176, in _find_and_load",
        "      File \"<frozen importlib._bootstrap>\", line 1147, in _find_and_load_unlocked",
        "      File \"<frozen importlib._bootstrap>\", line 690, in _load_unlocked",
        "      File \"/tmp/plugin_test/nonebot-plugin_orangedice-nonebot_plugin_orangedice/.venv/lib/python3.11/site-packages/nonebot/plugin/manager.py\", line 255, in exec_module",
        "        super().exec_module(module)",
        "      File \"<frozen importlib._bootstrap_external>\", line 940, in exec_module",
        "      File \"<frozen importlib._bootstrap>\", line 241, in _call_with_frames_removed",
        "      File \"/tmp/plugin_test/nonebot-plugin_orangedice-nonebot_plugin_orangedice/.venv/lib/python3.11/site-packages/nonebot_plugin_orangedice/__init__.py\", line 64, in <module>",
        "        plugin_config = Config.parse_obj(driver.config)",
        "      File \"/tmp/plugin_test/nonebot-plugin_orangedice-nonebot_plugin_orangedice/.venv/lib/python3.11/site-packages/pydantic/main.py\", line 1162, in parse_obj",
        "        return cls.model_validate(obj)",
        "      File \"/tmp/plugin_test/nonebot-plugin_orangedice-nonebot_plugin_orangedice/.venv/lib/python3.11/site-packages/pydantic/main.py\", line 596, in model_validate",
        "        return cls.__pydantic_validator__.validate_python(",
        "    \u001b[31m\u001b[1mpydantic_core._pydantic_core.ValidationError\u001b[0m:\u001b[1m 1 validation error for Config",
        "      Input should be a valid dictionary or instance of Config [type=model_type, input_value=Config(driver='fake', hos....timedelta(seconds=120)), input_type=Config]",
        "        For further information visit https://errors.pydantic.dev/2.9/v/model_type\u001b[0m"
    ],
    "load": false,
    "run": true,
    "version": "0.3.9",
    "config": null
}